src/pipecat/services/assemblyai.py

```diff
@@ -32,12 +32,12 @@ def __init__(
         self,
         *,
         api_key: str,
-        sample_rate: int = 16000,
+        sample_rate: int = 24000,
```

Comment:
This is an STT service, so we should leave at 16000.

Reply:
Fixed.